pkgs/desktops/gnome/default.nix

```diff
@@ -88,9 +88,7 @@ lib.makeScope pkgs.newScope (self: with self; {
 
   gnome-settings-daemon = callPackage ./core/gnome-settings-daemon { };
 
-  # Pantheon is using ubuntu and elementary's patchset due to reasons
-  # explained here -> https://github.com/elementary/greeter/issues/92#issuecomment-376215614
-  # Take note of "I am holding off on "fixing" this bug for as long as possible."
+  # Using 3.38 to match Mutter used in Pantheon
```

Comment:
As `light-locker` is no longer used in Pantheon 6, want to experiment a little bit in 2df0402e4bbf69e01a2051b7042981dd28666fd1 to see if everything works if the upstream gnome-settings-daemon is used.

Reply:
will test now on my system